bin/install-clj

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ex
+
+# Installs bin/clojure to bootstrap tools.deps installing on Jenkins
+# Use --verify to print dependencies
+
+CLJ_VERSION="1.10.1.489"
+# https://clojure.org/guides/getting_started#_installation_on_linux
+curl -O https://download.clojure.org/install/linux-install-$CLJ_VERSION.sh
```

Comment:
Not related to this PR really, but might be good to add mac-compatible set up options for local dev

Reply:
yep, good call

Reply:
Realized after looking closer there are brew install instructions, but really this script is not for local development, but for running it on jenkins automatically (which is Linux), but I left some todos and such to see if we can tighten it up later.